src/pygeosx/pygeosx_AcousticTest.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Solvers>
+    <!-- define the solver -->
+    <!-- we'll have to discuss the cflFactor with Randy -->
+    <!-- define the source coordinates -->
+    <!-- define the time source frequency -->
+    <!-- define the receiver coordinates -->
+    <AcousticSEM
+      name="acousticSolver"
+      cflFactor="0.25"
+      discretization="FE1"
+      targetRegions="{ Region }"
+      sourceCoordinates="{ { 1005.0, 1005.0, 1005.0 } }"
+      timeSourceFrequency="5.0"
+      receiverCoordinates="{ { 1105,1005, 1005 } }"
+      outputSeismoTrace="1"
+      dtSeismoTrace="0.05"/>
+
+  </Solvers>
+
+  <!-- hexahedral mesh generated internally by GEOSX -->
+  <!-- there is also an option to import external meshes -->
+  <Mesh>
+    <InternalMesh
+      name="mesh"
+      elementTypes="{ C3D8 }"
+      xCoords="{ 0, 2000 }"
+      yCoords="{ 0, 2000 }"
+      zCoords="{ 0, 2000 }"
+      nx="{ 20 }"
+      ny="{ 20 }"
+      nz="{ 20 }"
+      cellBlockNames="{ cb }"/>
+  </Mesh>
+
+  <Geometry>
+    <!-- <Box
+      name="zneg"
+      xMin="-0.01, -0.01, -0.01"
+      xMax=" 2000.01, 2000.01 , 0.01"/>
+    -->
+    <Box
+      name="zpos"
+      xMin="{-0.01, -0.01, 1999.99}"
+      xMax="{2000.01, 2000.01, 2000.01}"/>
+
+  </Geometry>
+
+
+  <Events
+    maxTime="0.2">
+    <!-- trigger the application of the solver -->
+    <!-- control the timestepping here with forceDt -->
+    <!-- (there are other ways to control the time step size) -->
+
+    <PeriodicEvent
+      name="solverApplications"
+      forceDt="0.05"
+      target="/Solvers/acousticSolver"/>
+    <!-- generate an output that can be read from VTK -->
+
+    <PeriodicEvent
+      name="waveFieldCollection"
+      timeFrequency="1.0"
+      targetExactTimestep="0"
+      target="/Tasks/waveFieldCollection" />
+
+
+    <PeriodicEvent
+      name="waveFieldOutput"
+      timeFrequency="1.0"
+      targetExactTimestep="0"
+      target="/Outputs/waveFieldOutput" />
+  </Events>
+
+  <NumericalMethods>
+    <FiniteElements>
+      <FiniteElementSpace
+        name="FE1"
+        order="1"/>
+    </FiniteElements>
+  </NumericalMethods>
+
+  <ElementRegions>
+    <CellElementRegion
+      name="Region"
+      cellBlocks="{ cb }"
+      materialList="{ nullModel }"/>
+  </ElementRegions>
+
+  <Constitutive>
+
+    <!-- add something here if needed -->
+    <NullModel
+      name="nullModel"/>
+  </Constitutive>
+
+  <FieldSpecifications>
+
+    <!-- here use FieldSpecification to define: -->
+
+
+    <!-- 2) The initial pressure field -->
+    <FieldSpecification
+      name="initialPressure"
+      initialCondition="1"
+      setNames="{ all }"
+      objectPath="nodeManager"
+      fieldName="pressure_n"
+      scale="0.0"/>
+
+    <FieldSpecification
+      name="initialPressure"
+      initialCondition="1"
+      setNames="{ all }"
+      objectPath="nodeManager"
+      fieldName="pressure_nm1"
+      scale="0.0"/>
```

Comment:
xml blocks should have unique names to avoid unexpected behavior during execution

Reply:
Also good to know thanks.